src/core/manager.h

```diff
@@ -233,7 +233,7 @@ struct Manager {
         Hashmap *unit_id_map;
         Hashmap *unit_name_map;
         Set *unit_path_cache;
-        usec_t unit_cache_mtime;
+        uint64_t unit_cache_timestamp;
```

Comment:
hmm, "timestamp" is not the best name if this is not actually a timestamp. Maybe `unit_cache_hashed_timestamp`,  or so would be a better name?

Reply:
I went with `_timestamp_hash`.